labels/en-US.json

```diff
@@ -239,6 +239,10 @@
   "friday": "Friday",
   "from": "from",
   "fullDesc": "Description",
+  "gatewayGiftCardDeleteEmail": "DO NOT DELETE THIS EMAIL",
+  "gatewayGiftCardEmailIntro": "Thank you for purchasing a {0} Gift Card. Do not delete this message -- to redeem the gift card you must use the gift card claim code below. You may want to print a copy of this message for easy reference later.",
+  "gatewayGiftCardNeedsCard": " You will need the card to complete your purchase" ,
```

Comment:
Text doesn't match what's in the jira issue. Should say `card number`.

Reply:
Changed the text and squashed commits that's why its saying outdated.